_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,361 @@
+  will now broadcast the onchain transaction 24 blocks before the HTLC
+  timeout, instead of 6 blocks before the timeout.
+
+- [LND #4018][] adds the ability to delay forwarding a payment (HTLC),
+  giving an external process the ability to review it and decide whether
+  to cancel it or continue forwarding it.  This is similar to [hold
+  invoices][topic hold invoices] but it applies to payments being routed
+  to a node's peers rather than received by the node itself.  Several
+  use cases are described in the PR---for example, one idea is to hold an
+  HTLC, detect that its next hop is offline (e.g. a mobile node), send a
+  notification to that node which will restart it, and then continue
+  relaying the HTLC.
+
+- [LND #4106][] adds a `getrecoveryinfo` RPC that returns information
+  about the progress of restoring from a backup.
+
+- [BIPs #933][] adds the [BIP339][] specification for transaction
+  relay announcement using Witness Transaction Identifiers (wtxids).
+  Currently, nodes announce new unconfirmed transactions to their peers
+  using the transaction's txid, which is a hash over the legacy fields
+  of the transaction.  The new fields used in segwit transactions are
+  not included in the txid, which was necessary to eliminate third-party
+  and counterparty transaction malleability.  However, during his June 2018 review
+  of segwit, Peter Todd [noticed][todd relay malleability] that
+  announcing transactions by their txid could allow nodes to modify the
+  segwit fields before relaying a transaction.  This couldn't be used to
+  steal money directly, but it did allow a relay node to mutate a valid
+  transaction into an invalid or unacceptable transaction without
+  changing its txid.
+
+    Before Todd's discovery, this was a problem: Bitcoin Core would
+    track the txids of invalid transactions it had recently seen and
+    refuse to download them again.  That meant a malicious node could
+    prevent any of its peers from downloading a valid segwit transaction
+    from any of their other peers by sending them a mutated version of
+    that transaction.  Arbitrary transaction censorship, even just at
+    the relay level, is bad by itself---but it has especially severe
+    consequences for time-sensitive protocols such as LN.
+
+    At the time of Todd's analysis, segwit was nearing release, so a
+    [quick fix][Bitcoin Core #8312] was implemented that prevents
+    Bitcoin Core from caching the rejection status of segwit
+    transactions that fail for the type of witness-related errors that
+    malicious nodes can inject.  This eliminated the issue, but it means
+    that Bitcoin Core uses more bandwidth than necessary when it
+    encounters segwit transactions that are invalid because of
+    accidental mistakes.  It may also complicate the development of new
+    relay protocols such as [package relay][topic package relay].
+
+    The long-term solution to the problem is that transactions should be
```

Comment:
"long-term" is used here (and "third-party is used thrice in this issue) but "long term", "medium term", "short term", and also "longer term" are used later on.